### ISSUE TITLE
Examples to highlight metrics, model-selection and convergence

### ISSUE THREAD

**Issue by mnarayan** [skggm/skggm#20]
What we now have compared to sklearn.covariance 
1. Additional metrics (quadratic loss, kl-loss) (add model selection loss tp+fp) ?
2. Additional methods for model selection/tuning parameters (ebic/bic and others coming)
3. Better convergence guarantees (no figures here, just examples that break GraphLasso)
4. What about timing benefits? (quic is supposed to be faster, big-quic definitely so)
5. Would love some examples that take advantage of the meta-estimators.  Even we if we reproduce something from the papers (if possible).
- [x] Example 1: Comparison of model selection procedures under different metrics
- [x] Example 2: Comparison of timing benefits for a range of (n,p) sizes
- [ ] Example 3: Compare number of iterations for convergence + convergence failure in ill-conditioned regimes
